### DISCUSSION TITLE
Problem

### DISCUSSION THREAD

**Discussion by Lucky390163** [libre-tube/LibreTube#5135] [General]
It is stuck on loading even though my internet connection is fast.Libre official servers are not working.

**Comment by Silther** [comment]
Did you try changing the instance?

**Reply by Lucky390163** [reply]
Yes, to official servers, but still not playing, but comments are loading fastly.

**Comment by M00NJ** [comment]
Try other insances from the list or turn off piped proxy temporarly.

**Comment by Lucky390163** [comment]
I don't use pipes proxy.

**Comment by M00NJ** [comment]
In that case, your only option is to try all the instances until you find a working one :/ 
Also, check that you have a good internet connection. YouTube is IP-banning instances. To avoid having to create new instances everytime that happens, some instances (especially the official ones) have started to use VPNs, wich slows things down.

**Comment by Lucky390163** [comment]
One more bug continue watch sectio shows 0 views please fix that
